GOVERNANCE.md

```diff
@@ -2,94 +2,171 @@
 
 # Governance
 
-This document outlines the governance model for Astro. This includes the contributor model, code review, merging, and the consequences and process for Code of Conduct violations.
+This document outlines the governance model for Astro. This includes the contributor model, code review process, PR merge process, and the consequences of Code of Conduct violations.
```

Comment:
(I honestly don’t know; just asking) should Contributor Model be a proper noun (i.e. referring to this entire document)?

Reply:
I guess so! Good catch